Before: 905ee37f0a855fb7d1f83563f8c2a6b1b549d8e414b5fc73831290d5d2e533c6
After: a3ec384945f966fd88ea63a40d223114dad9549488523ecfe17c859b02757373
Patch: 49054ca129c78ab1025a8ba4f4b4e67f1a44d84b6a52fe9a49da92f5496542f4
--- .rspec	2012-07-22 20:44:49.418916374 +1000
+++ /dev/null	2012-07-22 20:29:06.375709578 +1000
@@ -1,8 +0,0 @@
---color
-
---order random
-#--order random:12321 # Use a specific seed value
-
-#--format progress
---format documentation
-
--- bin/guard	2012-07-22 20:44:49.418916374 +1000
+++ /dev/null	2012-07-22 20:29:06.375709578 +1000
@@ -1,16 +0,0 @@
-#!/usr/bin/env ruby
-#
-# This file was generated by Bundler.
-#
-# The application 'guard' is installed as part of a gem, and
-# this file is here to facilitate running it.
-#
-
-require 'pathname'
-ENV['BUNDLE_GEMFILE'] ||= File.expand_path("../../Gemfile",
-  Pathname.new(__FILE__).realpath)
-
-require 'rubygems'
-require 'bundler/setup'
-
-load Gem.bin_path('guard', 'guard')
--- bin/rake	2012-07-22 20:44:49.418916374 +1000
+++ /dev/null	2012-07-22 20:29:06.375709578 +1000
@@ -1,16 +0,0 @@
-#!/usr/bin/env ruby
-#
-# This file was generated by Bundler.
-#
-# The application 'rake' is installed as part of a gem, and
-# this file is here to facilitate running it.
-#
-
-require 'pathname'
-ENV['BUNDLE_GEMFILE'] ||= File.expand_path("../../Gemfile",
-  Pathname.new(__FILE__).realpath)
-
-require 'rubygems'
-require 'bundler/setup'
-
-load Gem.bin_path('rake', 'rake')
--- bin/rspec	2012-07-22 20:44:49.418916374 +1000
+++ /dev/null	2012-07-22 20:29:06.375709578 +1000
@@ -1,16 +0,0 @@
-#!/usr/bin/env ruby
-#
-# This file was generated by Bundler.
-#
-# The application 'rspec' is installed as part of a gem, and
-# this file is here to facilitate running it.
-#
-
-require 'pathname'
-ENV['BUNDLE_GEMFILE'] ||= File.expand_path("../../Gemfile",
-  Pathname.new(__FILE__).realpath)
-
-require 'rubygems'
-require 'bundler/setup'
-
-load Gem.bin_path('rspec-core', 'rspec')
--- test/bootstrap/spec.rb	2012-07-22 20:44:49.418916374 +1000
+++ test/bootstrap/spec.rb	2012-07-22 21:27:56.063585834 +1000
@@ -1,5 +1,3 @@
 # encoding: utf-8
-$coverage_name = 'specs'
 require_relative 'all'
-require 'corvid/test/bootstrap/spec'
 
